build.sbt

```
@@ -63,12 +101,11 @@ val commonSettings: immutable.Seq[Def.Setting[_]] = List(
   ),
   organization := "com.gu",
   version := "1.0",
-  scalaVersion := "2.12.5",
+  scalaVersion := "2.12.18",
   scalacOptions ++= Seq(
     "-deprecation",
     "-encoding",
     "UTF-8",
-    "-target:jvm-1.8",
     "-Ypartial-unification",
```

It's not essential, but I quite like adding `-release:11` to the `scalacOptions`, as a way of sbt ensuring that it's being run with at least Java 11 or above:

```suggestion
    "-release:11",
    "-Ypartial-unification",
```

...the `-release:11` flag has the effect of ensuring that the generated class files are suitable for being run on a Java 11 runtime.